go.mod

```diff
@@ -104,4 +110,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
 
-replace github.com/openshift/assisted-service/models => github.com/openshift/assisted-service/models v0.0.0-20230831114549-1922eda29cf8
+replace (
+	github.com/openshift/assisted-service/api => github.com/openshift/assisted-service/api v0.0.0-20241008123739-b3cc2e3746d2 // release-4.17
```

Comment:
Why pin to `release-4.17` rather than `release-ocm-2.13` which actually maps to what is released in ACM?